ts/store/reducers/instabug/index.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * Instabug reducer
+ */
+
+import { combineReducers } from "redux";
```

Comment:
you don't need this file since it does nothing.
You have only 1 reducer so **combineReducers** is no needed
It can be deleted

Reply:
Ok